exporter/datadogexporter/metrics/utils.go

```diff
@@ -62,9 +63,15 @@ func NewCount(name string, ts uint64, value float64, tags []string) datadog.Metr
 }
 
 // DefaultMetrics creates built-in metrics to report that an exporter is running
-func DefaultMetrics(exporterType string, hostname string, timestamp uint64) []datadog.Metric {
+func DefaultMetrics(exporterType string, hostname string, timestamp uint64, buildInfo component.BuildInfo, withCollectorVersion bool) []datadog.Metric {
+	tags := []string{}
+
+	if withCollectorVersion {
+		tags = append(tags, "version:"+buildInfo.Version)
```

Comment:
Maybe we want to add the `buildInfo.Command` as a tag too? We add that on the metadata to know if the exporter is running on the contrib distro/the AWS distro/a custom distro and it could be useful here

Reply:
sure, do we have concerns about cardinality there? probably incredibly unlikely but could have some edge cases where there's a dynamic component

Reply:
Docs say this should be the *Executable file name, e.g. "otelcol"* so I would expect this to be a static string of that kind, what dynamic case did you have in mind?

Reply:
kk i think its fine, lets ship it